vertx-sockjs-service-proxy/src/test/resources/vertx-js/vertx-eventbus.js

```diff
@@ -308,6 +340,16 @@
     }
   };
 
+  EventBus.prototype.reconnectEnabled = function (enable) {
```

Comment:
Committed https://github.com/vert-x3/vertx-web/pull/660/commits/36c177eb405d9fcfb125750b9de5190def9fa5c5, although this is a breaking change, I highly doubt that anybody has ever used the pingEnabled method previously (it's definitely not common)